### PR TITLE
Release prep

### PR DESCRIPTION
**Pull request by apetrynet**
Preparing the ALE adapter repo for release of OpenTimelineIO 0.17.0 that ships without adapters

Feedback is welcome

### CODE REVIEW

**Comment by apetrynet** [comment]
@JeanChristopheMorinPerso Do you have any theory on why the windows test fails for python 3.11?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -24,34 +24,25 @@ jobs:
...
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
+          python -m pip install --upgrade pip cmake
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:41]: You don't need to install cmake. Cmake will be installed by https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/pyproject.toml#L5.

Comment (apetrynet) [reply .github/workflows/ci.yaml:41]: >It looks like a long path issue:

Oh, wow. How can we deal with that?

Comment (JeanChristopheMorinPerso) [reply .github/workflows/ci.yaml:41]: I'm not really sure how we could get rid of the issue...

**Comment by JeanChristopheMorinPerso** [comment]
@apetrynet can you try to add a little debugging step?

```
- name: Check LongPathsEnabled
  run: |
     (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
```
This will tell us if the long path mode is enabled or not.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -24,34 +24,32 @@ jobs:
...
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Check LongPathsEnabled
+        run: |
+          if [[ "${{ matrix.os }}" == "windows-latest" ]]; then
+            (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
+          fi
+        shell: bash
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:44]: We need poweshell technically.

Comment (apetrynet) [reply .github/workflows/ci.yaml]: Sorry, I'm not that familiar with this. I'll look up powershell logic.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -24,34 +24,32 @@ jobs:
...
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Check LongPathsEnabled
+        run: |
+          if [[ "${{ matrix.os }}" == "windows-latest" ]]; then
+            (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
+          fi
+        shell: pwsh
+
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:45]:
```suggestion
      - name: Check LongPathsEnabled
        if: ${{ matrix.os }} == "windows-latest"
        run: |
          (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
        shell: pwsh

```

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -24,34 +24,31 @@ jobs:
...
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Check LongPathsEnabled
+        if: ${{ matrix.os }} == "windows-latest"
+        run: |
+            (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
+        shell: pwsh
+
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:44]:
```suggestion
      - name: Check LongPathsEnabled
        if: ${{ matrix.os == "windows-latest }}
        run: |
            (Get-ItemProperty "HKLM:System\CurrentControlSet\Control\FileSystem").LongPathsEnabled
        shell: pwsh

```

File: .github/workflows/ci.yaml [hunk 5]
```diff
@@ -24,34 +24,30 @@ jobs:
...
+      - uses: actions/checkout@v4
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Check LongPathsEnabled
+        if: ${{ matrix.os == "windows-latest }}
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:40]:
```suggestion
        if: ${{ matrix.os == "windows-latest" }}
```

File: .github/workflows/ci.yaml [hunk 6]
```diff
@@ -24,34 +24,30 @@ jobs:
...
+      - uses: actions/checkout@v4
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Check LongPathsEnabled
+        if: ${{ matrix.os == "windows-latest" }}
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:40]:
```suggestion
        if: ${{ matrix.os == 'windows-latest' }}
```

File: .github/workflows/ci.yaml [hunk 7]
```diff
@@ -24,34 +24,30 @@ jobs:
...
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           if [[ "${{ matrix.otio-version }}" == "main" ]]; then
             pip install "git+https://github.com/AcademySoftwareFoundation/OpenTimelineIO.git"
-          elif [[ "${{ matrix.otio-version }}" == "extract_adapters" ]]; then
-            pip install "git+https://github.com/AcademySoftwareFoundation/OpenTimelineIO.git@${{ matrix.otio-version }}"
           else
-            pip install -V OpenTimelineIO==${{ matrix.otio-version }}
+            pip install -V OpenTimelineIO>=${{ matrix.otio-version }}
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:50]:
```suggestion
            pip install -vvv OpenTimelineIO==${{ matrix.otio-version }}
```

File: .github/workflows/ci.yaml [hunk 8]
```diff
@@ -24,34 +24,30 @@ jobs:
...
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           if [[ "${{ matrix.otio-version }}" == "main" ]]; then
             pip install "git+https://github.com/AcademySoftwareFoundation/OpenTimelineIO.git"
-          elif [[ "${{ matrix.otio-version }}" == "extract_adapters" ]]; then
-            pip install "git+https://github.com/AcademySoftwareFoundation/OpenTimelineIO.git@${{ matrix.otio-version }}"
           else
-            pip install -V OpenTimelineIO==${{ matrix.otio-version }}
+            pip install -vvv OpenTimelineIO==${{ matrix.otio-version }}
```
Comment (JeanChristopheMorinPerso) [inline .github/workflows/ci.yaml:50]:
```suggestion
            pip install -vvv OpenTimelineIO>=${{ matrix.otio-version }} --pre
```

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**